src/renderer/components/command-palette/command-container.tsx

```diff
@@ -29,8 +28,9 @@ export class CommandOverlay {
 }
 
 @observer
-export class CommandContainer extends React.Component<{cluster?: Cluster}> {
+export class CommandContainer extends React.Component {
   @observable.ref commandComponent: React.ReactElement;
+  @observable cluster = getHostedCluster();
```

Comment:
Why `getHostedCluster` vs passing it as a prop? This is an anti-pattern.

Reply:
Fixed, passing cluster id as a prop.